components/service/glean/src/test/java/mozilla/components/service/glean/storages/EventsStorageEngineTest.kt

```diff
@@ -154,4 +167,46 @@ class EventsStorageEngineTest {
         assertEquals("[[0,\"telemetry\",\"test_event_clear\",\"test_event_object\",null,null]]",
             snapshot.toString())
     }
+
+    @Test
+    fun `test sending of event ping when it fills up`() {
+        val server = MockWebServer()
+        server.enqueue(MockResponse().setBody("OK"))
+
+        EventsStorageEngine.clearAllStores()
+        val click = EventMetricType(
+            disabled = false,
+            category = "ui",
+            lifetime = Lifetime.Ping,
+            name = "click",
+            sendInPings = listOf("default"),
+            objects = listOf("buttonA")
+        )
+
+        val realClient = Glean.httpPingUploader
+        val testConfig = Glean.configuration.copy(
```

Comment:
Instead of doing this, we can pass in the default `with val testConfig = Configuration(...)`, leaving the definitions untouched.

Reply:
I'm not sure I follow...

Reply:
Yeah, don't bother about this. It was related to [this one](https://github.com/mozilla-mobile/android-components/pull/1572#discussion_r241403006), and it doesn't make sense :)